packages/language-server/src/server.ts

```diff
@@ -98,32 +98,51 @@ connection.onCompletion(doAutoCompletion(documents, neo4jSchemaPoller));
 
 connection.onDidChangeConfiguration(
```

Comment:
I need to remove this - at the moment, it remains so that the existing e2e tests work 

I need to update the e2e tests so that they create a `Connection` and connect to that, rather than relying on the settings